strings.go

```diff
@@ -306,3 +315,51 @@ func keyStmt(key []byte) *ast.GenDecl {
 		}},
 	}
 }
+
+func strConstBlacklist(node ast.Node, info *types.Info, blacklist map[types.Object]struct{}) {
+	strType := types.Typ[types.String]
+	untypedStr := types.Typ[types.UntypedString]
+
+	constCheck := func(node ast.Node) bool {
```

Comment:
I think a better name here could be `blacklistObjects`, since the point of the func is to blacklist all objects referenced anywhere within an expression.

Reply:
OK